src/runners/handlers/slack.py

```diff
@@ -67,19 +66,31 @@ def handle(
     slack_token = vault.decrypt_if_encrypted(slack_token_ct)
 
     sc = SlackClient(slack_token)
-
     # otherwise we will retrieve email from assignee and use it to identify Slack user
     # Slack user id will be assigned as a channel
 
     title = alert['TITLE']
 
     if recipient_email is not None:
-        result = sc.api_call("users.lookupByEmail", email=recipient_email)
+        if isinstance(recipient_email, str):
+                user_object = sc.api_call("users.lookupByEmail", email=recipient_email)
```

Comment:
I can do this before merging, but would save us an extra step if you could install Python Black in your text editor and fix e.g. this indentation to be 4 spaces.